src/Trakx.Ubiquity.ApiClient.Tests/Integration/ProtocolsClientTests.cs

```diff
@@ -15,11 +15,11 @@ public ProtocolsClientTests(UbiquityApiFixture apiFixture, ITestOutputHelper out
         _client = ServiceProvider.GetRequiredService<IProtocolsClient>();
     }
 
-    [Fact]
+    [Fact(Skip = "This test is not working. Response: {'type':'no-endpoint','code':16387,'title':'No Such Endpoint','status':404,'detail':'No API endpoint is available at this path'}")]
```

Comment:
@monsieurleberre this test is failing. It seems the endpoint doesn't exist anymore. Should we update the `openApi3.yaml` in the next PR?

Reply:
For now, it is okay to keep that way.
Created task TKX960.